README.md

```diff
@@ -62,6 +62,10 @@ Here is an example for the dialplan:
     exten => _X.,1,Dial(Dongle/r1/${EXTEN})
     exten => _X.,n,Hangup
 
+    exten => _X.,n,DongleSendSMS(dongle0,${EXTEN},"Please call me",1440,yes)
+
+    exten => _X.,n,DongleSendUSSD(dongle1,*#123#)
+
```

Comment:
Yup. If you haven't committed your changes already:
```
git commit . --amend --date "$(date -R)"
git push -f
```
If you have committed already:
```
git rebase -i HEAD~2
<now change the 2nd "pick" to "f">
git commit  --amend --date "$(date -R)"
<save/quit, will update the date only>
git push -f
```